PowerPointLabs/PowerPointLabs/ELearningLab/Views/ELearningWorkspaceViews/ELearningLabMainPanel.xaml.cs

```diff
@@ -91,26 +92,27 @@ public void ListViewItemPropertyChanged(object sender, PropertyChangedEventArgs
         /// </summary>
         public void ReloadELearningLabOnSlideSelectionChanged()
         {
-            PowerPointSlide _slide = this.GetCurrentSlide();
+            slide = this.GetCurrentSlide();
+            if (slide == null)
+            {
+                slideId = -1;
+                SetupSpinnerOnLoadingBegins("No slide is selected");
```

Comment:
Could all these constant strings be placed in a ELearningText script file with constants?

Reply:
Thanks for the suggestion! Removed magic strings.